source/index.html

```diff
@@ -766,6 +775,99 @@ <h3 class='title'>Pricing</h3>
 				<button id='question-pricing-button'>Details</button>
 			</div>
 		</section>
+		
+		 <section id='addonsbanner'>
+			<style>
+				#addonsbanner {
+					grid-column: 1;
+					grid-row: 13;
+					display: flex;
+					justify-content: space-around;
+					align-items: stretch;
+					row-gap: 50px;
+					padding-top: 20px;
+					padding-bottom: 20px;
+				}
+
+				
+				
+				@media screen and (max-width: 680px) {
+					#addonsbanner {
+						flex-direction: column;
+						align-items: center;
+					}
+					#addonsbanner > .addonsbanner {
+						grid-column: 1;
+						grid-row: 1;
+
+				}
+
+			</style>
+
+                        <div class='addonsbanner'>
+
+			<h1> + ADD-ONS </h1>
+	
+			</div>
+
+                       
+                        </section>
+
+
+
+
+                       <section id='addons'>
+
+			<style>
+				#addons {
+					grid-column: 1;
+					grid-row: 14;
+					display: flex;
+					justify-content: space-around;
+					align-items: stretch;
+					row-gap: 50px;
+					padding-top: 20px;
+					padding-bottom: 50px;
+				}
+				#addons > .pricing-specs > .freebie {
+					flex-grow: 1;
+				}
+				@media screen and (max-width: 680px) {
+					#addons {
+						flex-direction: column;
+						align-items: center;
+					}
+					#addons> .pricing-specs.month {
+						grid-column: 1;
+						grid-row: 1;
+					
+					              }
+				}
+			</style>
+
+				
+			
+			<div class='pricing-specs month'>
+				<div class='amount'>$100</div>
+				<div class='per'><span>Per</span></div>
+				<div class='unit'>Month</div>
+				<div class='freebie'>(Chat)</div>
+				<button id='addonchat'>Details</button>
+			</div>
+			<div class='pricing-specs month'>
+				<div class='amount'>$200</div>
+				<div class='per'>Per</div>
+				<div class='unit'>Month</div>
+				<div class='freebie'>(Non-chat)</div>
+				<button id='addonnonchat'>Details</button>
+                         </div>
+			
+
+              </section>
+		
+	
+		
```

Comment:
Formatting cleanup.  As mentioned above, we should try to maintain a consistent style across the document, which includes tabs for indentation, blank lines only between sections, etc.
```suggestion
			</style>
			]<div class='addonsbanner'>
				<h1> + ADD-ONS </h1>
			</div>
		</section>

		<section id='addons'>
			<style>
				#addons {
					grid-column: 1;
					grid-row: 14;
					display: flex;
					justify-content: space-around;
					align-items: stretch;
					row-gap: 50px;
					padding-top: 20px;
					padding-bottom: 50px;
				}
				#addons > .pricing-specs > .freebie {
					flex-grow: 1;
				}
				@media screen and (max-width: 680px) {
					#addons {
						flex-direction: column;
						align-items: center;
					}
					#addons> .pricing-specs.month {
						grid-column: 1;
						grid-row: 1;
					}
				}
			</style>
			<div class='pricing-specs month'>
				<div class='amount'>$100</div>
				<div class='per'><span>Per</span></div>
				<div class='unit'>Month</div>
				<div class='freebie'>(Chat)</div>
				<button id='addonchat'>Details</button>
			</div>
			<div class='pricing-specs month'>
				<div class='amount'>$200</div>
				<div class='per'>Per</div>
				<div class='unit'>Month</div>
				<div class='freebie'>(Non-chat)</div>
				<button id='addonnonchat'>Details</button>
			</div>
		</section>
```